### PR TITLE
 [api] Move ProjectLogRotate to 1 event per job 

### PR DESCRIPTION
**Pull request by Ana06**
The process for creating a ProjectLogEntry from an event is now:

1. After an event is created, create a delayed job with the event's payload in the DJ.
2. When the DJ runs:
2.1. create a ProjectLogEntry based on the payload in the DJ.
2.2. update the event.project_logged = true

I also introduced a rake task to be able to run this manually.

I regenerated the Rubocop todo file. There is a Rubocop bug, that's why I needed to modify the Rubocop configuration as well. I could add this new offense to the `.rubocop.yml` instead of everything to the todo, but that would mean we do not plan to solve the offense, and I would say this is not the case.

### CODE REVIEW

**Comment by Ana06** [comment]
[(8) P8: Move ProjectLogRotate to 1 event per job](https://trello.com/c/vOh0oyHu/619-8-p8-move-projectlogrotate-to-1-event-per-job)

File: src/api/app/models/event.rb [hunk 1]
```diff
@@ -1,2 +1,20 @@
 module Event
+  PROJECT_CLASSES = ["Event::CommentForProject",
```
Comment (evanrolfe) [inline src/api/app/models/event.rb]: Nice solution!

File: src/api/app/jobs/project_log_rotate_job.rb [hunk 2]
```diff
@@ -1,41 +1,9 @@
 class ProjectLogRotateJob < ApplicationJob
```
Comment (evanrolfe) [inline src/api/app/jobs/project_log_rotate_job.rb]: [Nitpick] Small point I know but can you please change the name to something like `CreateProjectLogEntry`? The name doesnt make sense anymore..

**Comment by eduardoj** [comment]
I deleted a comment by mistake. Sorry!

> @evanrolfe : [Nitpick] Small point I know but can you please change the name to something like CreateProjectLogEntry? The name doesnt make sense anymore..

**Comment by ChrisBr** [comment]
[Fix Package Activity SQL](https://trello.com/c/SYa0glOY/1269-fix-package-activity-sql)

**Comment by Ana06** [comment]
@mdeniz why `Test Suit`? 🤔

**Comment by mdeniz** [comment]
Was the labeling bot (using my credentials), you touched a spec file in the PR... so was labeled as Test Suite. I will remove the label, so, don't worry.

**Comment by evanrolfe** [comment]
Btw @Ana06 the specs are failing because you changed the behaviour of the ProjectLogRotateJob but you didn't update the specs. Its not because there is anything wrong with your code I think..

**Pull request merged**